### PR TITLE
This Message Will Self-Destruct

### PR DESCRIPTION
**Pull request by Shadowmech88**
Adds explosive paper.
Once something is written on explosive paper, when it is read, it will notify the reader that it will self-destruct after a set amount of seconds, and will explode once that time is up.
The explosion is small enough to not cause a breach, though it will still hurt.

The default time is five seconds, though it can be varedited/subtyped to change that as needed.

### CODE REVIEW

File: code/modules/paperwork/explosive_paper.dm [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+			detonate()
+	..()
+
+/obj/item/weapon/paper/explosive/proc/detonate()
+	if(!detonating)
+		if(info)
+			info += "<BR><BR>THIS MESSAGE WILL SELF-DESTRUCT IN [fuse_time/10] SECONDS"
+			updateinfolinks()
+			spawn(fuse_time)
+				detonating = TRUE
```
Comment (Intigracy) [inline code/modules/paperwork/explosive_paper.dm]: move above the spawn()

**Review by Intigracy — CHANGES_REQUESTED** [review]

**Comment by Intigracy** [comment]
wait I didn't mean to do that
oops

**Comment by Intigracy** [comment]
Make a different PR if you planned to add it somewhere.

**Comment by CptWad** [comment]
wew
How about 2 telecrystals? While you are at it increase the decoys to 2 crystals too since you can make like 20 fucking decoys of yourself

**Comment by Blithering** [comment]
Making twenty decoys of yourself is the best use for your telecrystals though.

**Pull request merged**